lib/Service/OauthService.php

```diff
@@ -48,17 +48,11 @@ public function __construct(ClientMapper $clientMapper,
 	}
 
 	/**
-	 * @param string $name
-	 * @param string $redirectUri
-	 * @return array<mixed>
+	 * @param string $secret
+	 * @param string $nextcloudVersion
+	 * @return string
 	 */
-	public function createNcOauthClient(string $name, string $redirectUri): array {
-		$clientId = $this->secureRandom->generate(64, self::validChars);
-		$client = new Client();
-		$client->setName($name);
-		$client->setRedirectUri(sprintf($redirectUri, $clientId));
-		$secret = $this->secureRandom->generate(64, self::validChars);
-		$nextcloudVersion = OC_Util::getVersionString();
+	public function getHashedOrEncryptedSecretBasedOnNextcloudVersions(string $secret, string $nextcloudVersion): string {
```

Comment:
```suggestion
	public function hashOrEncryptSecretBasedOnNextcloudVersion(string $secret, string $nextcloudVersion): string {
```

`get` sounds like we are reading it from somewhere, what is not true